packages/postgres-database/src/simcore_postgres_database/migration/versions/83f9d2a470ef_add_cluster_id_in_comp_runs.py

```diff
@@ -0,0 +1,38 @@
+"""add cluster id in comp_runs
+
+Revision ID: 83f9d2a470ef
+Revises: dd8220be55ad
+Create Date: 2021-08-31 20:02:45.709839+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "83f9d2a470ef"
+down_revision = "dd8220be55ad"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("comp_runs", sa.Column("cluster_id", sa.BigInteger(), nullable=True))
```

Comment:
MINOR: update the tables ERD (Entity Relationship Diagram) by 
```command line
cd packages/postgres-database
make auto-doc
```

Reply:
will try on linux box.. it fails in wsl... will check maybe later why

Reply:
@pcrespov I will not update it since I'm not able to run it.
first I had to ```sudo apt install graphviz-dev``` and now I'm still getting an error
```bash
python scripts/create_erd.py
Traceback (most recent call last):
  File "/home/sanderegg/dev/github/osparc-simcore/.venv/lib/python3.8/site-packages/pygraphviz/agraph.py", line 1344, in _get_prog
    runprog = self._which(prog)
  File "/home/sanderegg/dev/github/osparc-simcore/.venv/lib/python3.8/site-packages/pygraphviz/agraph.py", line 1800, in _which
    raise ValueError(f"No prog {name} in path.")
ValueError: No prog dot in path.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "scripts/create_erd.py", line 29, in <module>
    main()
  File "scripts/create_erd.py", line 25, in main
    render_er(metadata, str(output))
  File "/home/sanderegg/dev/github/osparc-simcore/.venv/lib/python3.8/site-packages/eralchemy/main.py", line 236, in render_er
    intermediary_to_output(tables, relationships, output)
  File "/home/sanderegg/dev/github/osparc-simcore/.venv/lib/python3.8/site-packages/eralchemy/main.py", line 75, in intermediary_to_schema
    graph.draw(path=output, prog='dot', format=extension)
  File "/home/sanderegg/dev/github/osparc-simcore/.venv/lib/python3.8/site-packages/pygraphviz/agraph.py", line 1596, in draw
    data = self._run_prog(prog, args)
  File "/home/sanderegg/dev/github/osparc-simcore/.venv/lib/python3.8/site-packages/pygraphviz/agraph.py", line 1360, in _run_prog
    runprog = r'"%s"' % self._get_prog(prog)
  File "/home/sanderegg/dev/github/osparc-simcore/.venv/lib/python3.8/site-packages/pygraphviz/agraph.py", line 1346, in _get_prog
    raise ValueError(f"Program {prog} not found in path.")
ValueError: Program dot not found in path.
make: *** [Makefile:109: auto-doc] Error 1
```

Reply:
will put in in a container :-)